### ISSUE TITLE
pkg-config file missing ?

### ISSUE THREAD

**Issue by mrechte** [fedora-infra/fedora-packages#132]
Hello,
I don't know if the right place to post this, if not, sorry about that.
The -devel package does not include a pkg-config file.
Is that an issue ?

Thanks

**Comment by ralphbean** [comment]
I don't think I understand. Can you provide an example @mrechte?

**Comment by mrechte** [comment]
Trying to add the library to a project with anjuta IDE, I could not locate the library despite having installed the package. Information is here: https://developer.gnome.org/anjuta-manual/stable/project-manager-module-new.html.fr

**Comment by mrechte** [comment]
It seems I am in the wrong place. This issue if for a fedora package (dcmtk).

**Comment by ralphbean** [comment]
Ah, sorry.  You have filed this bug in the wrong place.  This would be better filed against the `anjuta` package I think.  I found a place to file that by following this process.

I searched for anjuta here:

https://apps.fedoraproject.org/packages/s/anjuta

I then clicked on the 'bugs' tab to get me here:

https://apps.fedoraproject.org/packages/anjuta/bugs

And then clicked on the "open a new bug" button which got me here:

https://bugzilla.redhat.com/enter_bug.cgi?product=Fedora&version=rawhide&component=anjuta

If you provide them with the link to the documentation there like you did here, I think that will help the developers figure out what's going on.
